browser/src/control/Control.UIManager.js

```diff
@@ -95,17 +95,27 @@ L.Control.UIManager = L.Control.extend({
 		this.map.fire('darkmodechanged');
 	},
 
-	setCanvasColorAfterModeChange: function() {
+	setCanvasColorAfterModeChange: function(nColor) {
 		if (app.sectionContainer) {
 			app.sectionContainer.setBackgroundColorMode(false);
-			app.sectionContainer.setClearColor(window.getComputedStyle(document.documentElement).getPropertyValue('--color-canvas'));
+			if (nColor == undefined )
+				nColor = window.getComputedStyle(document.documentElement).getPropertyValue('--color-canvas');
+			app.sectionContainer.setClearColor(nColor);
 			//change back to it's default value after setting canvas color
 			app.sectionContainer.setBackgroundColorMode(true);
 		}
 	},
 
 	invertBackground: function() {
 		app.socket.sendMessage('uno .uno:InvertBackground');
+		if (this.map.getDocType() == 'spreadsheet') {
+			var lightCanvasColor = '#f5f5f5'; //color-palette.css --color-canvas
+			var darkCanvasColor = '#141414'; //color-palette-dark.css --color-canvas
```

Comment:
We need in the future make it somehow shared...

Reply:
check: https://github.com/CollaboraOnline/online/blob/bada0174470651999b25c2edb619093702143fb5/browser/src/canvas/CanvasSectionContainer.ts#L254

can we programatically get this value? not hardcode?

Reply:
Szymon getCompuptedStyle gets the current color of --color-canvas, but i need to know light mode canvas colors even editor in dark mode.

Reply:
For me it is important to have this value (color) define din one place. So on change we don't need to look for many occurences

Reply:
It worked szymon, thanks